src/main/java/teammates/ui/output/FeedbackSessionSubmissionStatus.java

```diff
@@ -11,9 +11,9 @@ public enum FeedbackSessionSubmissionStatus {
     NOT_VISIBLE,
 
     /**
-     * Feedback session is visible to view but not open for submission.
+     * Feedback session is visible to view but not yet open for submission.
      */
-    VISIBLE_NOT_OPEN,
```

Comment:
Understand why you want to change this, but now is not the time to change API definitions. These definitions are propagated to the frontend as well. Those will have to updated too.

In fact, we try not to touch these especially during peak periods (typical school semester time). Updating these will cause a small disruption due to the way the service worker works (using a cached version and only updating the frontend on refresh).

Reply:
I have reverted the changes for this, though it did cause me some confusion when implementing isClosed and isOpened and I feel this term should be eventually updated to include the 'yet'.